Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml

```diff
@@ -10,4 +10,6 @@
     BoxBottle: 3
     ClothingEyesHudMedical: 2
     ClothingEyesEyepatchHudMedical: 2
-
+  contrabandInventory:
+    BrutepackAdvanced1: 1
```

Comment:
Strong disagree with this, even just one will have a tider foaming at the mouth to get.

Reply:
Originally this was a defib before I realized how awful that would be, even when they only cost like, 3 steel. I replaced it with these 1-ofs of the advanced topicals as an emergency med you can hack for (with 3 NanoMed Plus per medbay, that's 3 of each), and as a way for doctors to teach interns about them and demonstrate their use.

I might just need to scrap this one all together.

Reply:
Swapped for a singular apple. I think the joke tells itself.